Do not include TZ information in queries

walro opened the pull request:
Same as https://github.com/twingly/twingly-search-api-ruby/pull/69.

Re-record fixtures to get fixtures without published time bugs.

walro committed:
Do not include TZ information in queries

Same as https://github.com/twingly/twingly-search-api-ruby/pull/69.

Re-record fixtures to get fixtures without published time bugs.

walro commented on tests/fixtures/vcr_cassettes/search_without_valid_api_key.json:
```diff
@@ -1,43 +1 @@
-  "recorded_with": "betamax/0.8.0"
-}
\ No newline at end of file
+{"http_interactions": [{"request": {"body": {"string": "", "encoding": "utf-8"}, "headers": {"Connection": "keep-alive", "Accept-Encoding": "gzip, deflate", "Accept": "*/*", "User-Agent": "Twingly Search Python Client/2.1.0"}, "method": "GET", "uri": "https://api.twingly.com/blog/search/api/v3/search?q=something&apiKey=wrong"}, "response": {"body": {"string": "<?xml version=\"1.0\" encoding=\"utf ...
```
hmm I should probably update my betamax so get nicer output :) *cough* #11

walro replied:
didn't matter, maybe it's Windows vs macOS thing.

walro committed:
Re-record Betamax fixtures

… with newer betamax to get nicer output

roback commented on tests/test_query.py:
```diff
@@ -58,14 +58,14 @@ def test_query_should_add_start_date(self):
         q.start_time = self.datetime_with_timezone(datetime(2012, 12, 28, 9, 1, 22), "UTC")
         query_string = q.build_query_string()
-        self.assertEqual(query_string, "spotify start-date:2012-12-28T09:01:22Z")
+        self.assertEqual(query_string, "spotify start-date:2012-12-28 09:01:22")
```
Shouldn't it be `"spotify start-date:\"2012-12-28 09:01:22\""` (surrounded by quotes) now that the timestamp contains a space?

walro replied:
I don't think, the request sent will actually include %20 (or +).  Also take a look at what we had before: 4e4ef2bf116deb2b33dd473e658f16b228833e96

walro replied:
But yeah, I see our docs includes quoting, but I don't think it's necessary.

walro replied:
Hmm, investigated some more. Seems like you are right. We probably have this bug in most of clients. Nice catch.

roback replied:
Adding a `T` between the date and the time parts works as well.

walro replied:
> Adding a T between the date and the time parts works as well.

Yes, seems to be working. I am thinking we should possibly update our developer site to prefer this over quoting. Should be easier for customers and also easier for us when reading logs.

walro committed:
Include T in timestamps

See
https://github.com/twingly/twingly-search-api-python/pull/38#discussion_
r144828545

walro committed:
Too many T:s

With great search and replace comes great responsibility.
